native/packages/react-native-bpk-component-banner-alert/src/__snapshots__/BpkBannerAlert-test.ios.js.snap

```diff
@@ -302,14 +304,15 @@ exports[`iOS BpkBannerAlert should render correctly dismissable 1`] = `
               "lineHeight": 18,
             },
             Object {
+              "color": "rgb(82, 76, 97)",
               "flex": 1,
               "flexGrow": 1,
               "paddingLeft": 4,
             },
           ]
         }
       >
-        Successful alert.
+        Neutral alert.
```

Comment:
This snapshot looks weird, if Neutral has been added, how come the diff has `successful` replaced with neutral and no new tree for the one added?

Reply:
It would seem that the tests were never creating snapshots for anything other than the first component.

Have wrapped them all inside a view so that they actually render!